sdk/storage/storage-blob/.nycrc

```diff
@@ -21,6 +21,7 @@
       "html",
       "cobertura"
     ],
+    "exclude-after-remap":false,
```

Comment:
This is required since nyc is updated from 13.3.0 to 14.0.0.
[Breaking Change]
https://github.com/istanbuljs/nyc/blob/master/CHANGELOG.md
https://github.com/istanbuljs/nyc#source-map-support-for-pre-instrumented-codebases

Reply:
OK, how does coverage report look like? Do covered lines point to TS source code or compiled JS codes?

Reply:
Covered lines would point to the typescript source code.

**Example -** 
Coverage output-html for (only) one test -


![image](https://user-images.githubusercontent.com/10452642/57333733-82b73a80-70d2-11e9-949a-703b87f43df3.png)
![image](https://user-images.githubusercontent.com/10452642/57333799-b72af680-70d2-11e9-8ca9-079e40081860.png)